### DISCUSSION TITLE
200

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#179] [General]
# 潮流周刊第200期 - 云海日落

封面图拍摄于挺久之前在国外坐飞机拍到的一张云海落日的照片，玩周刊的一个很大的用途就是让我多去体验不一样的生活，多去看看外面的世界，居然一不小心潮流周刊都 200 期了，希望你也过得开心。

https://weekly.tw93.fun/posts/200-%E4%BA%91%E6%B5%B7%E6%97%A5%E8%90%BD

**Comment by ChengWeiDong** [comment]
200 期了，差不多追了有 100 多期，作为设计师来说也是很棒，每周一固定要阅读的周刊之一，再来 200 期哈哈哈哈

**Reply by xxholly32** [reply]
同上~

**Reply by lovingchina** [reply]
+1

**Reply by tw93** [reply]
哈哈 谢谢喜欢

**Comment by Vainycos** [comment]
打卡200期。一起加油，天天快乐

**Reply by tw93** [reply]
加油加油💪

**Comment by zyzhang0417** [comment]
坚持两百期，非常不容易；
我来打个卡，留个小标记。

**Comment by FtdijsJin** [comment]
打卡200，做大做强hhh，ヾ(◍°∇°◍)ﾉﾞ

**Comment by JulyFinal** [comment]
恭喜200期。潮流周刊真的是我看过最舒服的期刊了，兼顾了潮流和自然hhh

**Comment by FelixLyfe** [comment]
第200期，到此一游

**Comment by desmnmao** [comment]
热爱生活，期待下一个200期

**Comment by Hakunassan** [comment]
真牛啊 真牛啊！

**Comment by 55utah** [comment]
打卡打卡 200期 ～

**Comment by xh-0** [comment]
打开200期，每周上班的快乐源泉之一 ~~

**Comment by wenzhuo4657** [comment]
“你看到什么，听到什么，做什么，和谁在一起，有一种，从心灵深处，满溢出来的不懊悔，也不羞耻的平和与喜悦。”
这话讲的真棒，过了好几天突然想起来，很有味道的一句话

**Comment by qbmiller** [comment]
还有类似期刊推荐吗。 阮一峰除外

**Comment by Jinxhao18** [comment]
蓝胖子奶粉一个人喝不完，保质期好像是一个月，独居者慎重。

**Comment by treeechoo** [comment]
200期打卡~~
